google-cloud-os_login/MIGRATING.md

```diff
@@ -88,11 +88,11 @@ Google::Cloud::OsLogin::V1::OsLoginService::Client.configure do |config|
 end
 ```
 
-Defaults for certain configurations can be set for all Redis versions
+Defaults for certain configurations can be set for all OSLogin versions
```

Comment:
Shouldn't the name of the product be "OS Login" everywhere in this document, rather than "OSLogin"?

Reply:
Yep, good call.